### DISCUSSION TITLE
GitHub app does not detect Vivaldi browser on Android

### DISCUSSION THREAD

**Discussion by JoakJohaTech** [community/community#63292] [Mobile]
I use GitHub app on Android and I tried to login to codepen.io with Vivaldi web browser app with GitHub login option. When I get redirected from vi Vivaldi to the GitHub app. The option for what browser I want to use inside the GitHub app does not include vivaldi, it only includes chrome and chrome beta. Can the GitHub developers add vivaldi?

**Comment by alcere** [comment]
Hi @forwardbutton, we are sorry for this inconvenience! I will create an internal issue to fix this! thanks!
